### DISCUSSION TITLE
Trust a self-signed cert

### DISCUSSION THREAD

**Discussion by lmeerwood** [qutebrowser/qutebrowser#5873] [Q&A] [answered]
Hey, I'm running a VM that is running a web server that has a self-signed certificate. Whenever I try access the ip address it fails and says:

```
This site can’t be reached 
The web page at https://25.1.1.4/ might be temporarily down or it may have moved permanently to a new web address.
ERR_CERT_INVALID
```
With Firefox I can bypass the issue by accepting the invalid cert, and on chrome I can type out "thisisunsafe". Is there someway to bypass the issue on qutebrowser?

**Comment by toofar** [comment, marked as answer]
It looks like the `content.ssl_strict` setting supports URL patterns. So something like `set -u https://25.1.1.4/* content.ssl_strict false` may work. Our putting a pem in /etc/ssl/certs/.

**Reply by The-Compiler** [reply]
IMHO adding the certificate to the trusted certificates is the proper solution for this (for any browser, not just qutebrowser) - point 20 in the [FAQ](https://github.com/qutebrowser/qutebrowser/blob/master/doc/faq.asciidoc) about client certificates should also apply to self-signed server certificates, I think.
